packages/web-app-admin-settings/src/components/Users/UsersList.vue

```diff
@@ -7,6 +7,30 @@
       :label="$gettext('Filter users')"
       autocomplete="off"
     />
+    <div class="oc-flex oc-flex-middle oc-ml-m oc-mb-m oc-mt-m">
+      <div class="oc-mr-m oc-flex oc-flex-middle">
+        <oc-icon name="filter-2" fill-type="line" />
+        <span v-text="$gettext('Filter:')" />
+      </div>
+
+      <!-- TODO: just for demo purposes, remove later -->
+      <item-filter
+        filter-name="Users"
+        :items="users"
+        :show-filter="true"
+        :allow-multiple="true"
+        display-name-attribute="displayName"
+        :filterable-attributes="['displayName']"
+      >
+        <template #image="{ item }">
+          <avatar-image :width="32" :userid="item.id" :user-name="item.displayName" />
+        </template>
+        <template #item="{ item }">
+          <div v-text="item.displayName" />
```

Comment:
Would be nice if you could use `mark.js` (like we already do in other places) to highlight the filter term in the found results. Means, the filter term needs to be injected from the slot into the template.